cmd/kubeadm/app/util/config/upgradeconfiguration_test.go

```diff
@@ -104,3 +107,244 @@ func TestDocMapToUpgradeConfiguration(t *testing.T) {
 		})
 	}
 }
+
+func TestLoadUpgradeConfigurationFromFile(t *testing.T) {
+	tmpdir, err := os.MkdirTemp("", "")
+	if err != nil {
+		t.Fatalf("Couldn't create tmpdir: %v", err)
+	}
+	defer func() {
+		if err := os.RemoveAll(tmpdir); err != nil {
+			t.Fatalf("Couldn't remove tmpdir: %v", err)
+		}
+	}()
+	filename := "kubeadmConfig"
+	filePath := filepath.Join(tmpdir, filename)
+	options := LoadOrDefaultConfigurationOptions{}
+
+	tests := []struct {
+		name         string
+		cfgPath      string
+		fileContents string
+		want         *kubeadmapi.UpgradeConfiguration
+		wantErr      bool
+	}{
+		{
+			name:    "Config file not exists",
+			cfgPath: "tmp",
+			want:    nil,
+			wantErr: true,
+		},
+		{
+			name:         "Config file is not yaml format",
+			cfgPath:      filePath,
+			want:         nil,
+			fileContents: "Config file is not yaml format",
```

Comment:
```suggestion
			fileContents: "some-text",
```

use some basic text instead of the test case `name`